### ISSUE TITLE
When the keyboard focus is on any list item in the "Select an Azure ML job Schema" list and when user press Shift+tab key, JAWS/NVDA does not announce anything: A11y_VS Code_Appling filter on subscription_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#245224]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.3.1; #FTP; #Screen Reader; #NVDA; #JAWS; #Element:Combobox; #STP;

## Environment Details:
Application: VS Code
Visual studio code version: 1.96.3
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
JAWS version: 2024.2312.53
NVDA Version: 2024.4.1

## Repro Steps

1. Turn on Jaws/NVDA screen reader
2. Launch the VS Code.
3. Install the Azure Machine Learning extension and sign in.
4. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
5. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
6. The Azure panel will now be active next to the toolbar.
7. Tab until the Machine Learning dropdown.
8. TAB till "Create job" button and press ENTER key.
9. Focus will be on "Select an Azure ML job Schema" combo box and.
10. Now press "TAB" than SHIFT + TAB.
11. Observer that JAWS/NVDA does not announce any thing when keyboard focus is on any list item of "Select an Azure ML job Schema" list and pressing shift +TAB key

## Actual Experience.
When the keyboard focus is on any list item in the "Select an Azure ML job Schema" list and the user presses the Shift+Tab key, the keyboard focus moves to the "Select an Azure ML job Schema" edit field. However, the screen reader focus does not shift, so JAWS/NVDA does not announce anything. Now, pressing the Shift key moves the screen reader focus to the edit field.

### Similar issue observed with following scenarios.

**Scenario1**: Creating a job Select an Azure ML job Schema

1. TAB till create job button and press ENTER key
2. TAB till Combo box and press TAB and then SHIF + TAB

**Scenario2**: Setting Default Workspace

1. Open the Command Palette via the keyboard command Ctrl + Shift + P 
2. Type "Set default workspace" and select the option from the dropdown list called "Azure ML: Set Default Workspace") and press enter
3. A list should appear that says "Select a subscription"
4. Use the up/down arrows to navigate the list until "AML V1 Personal 2" is selected, and press enter
5. A list should appear that says "Select a workspace"
6. TAB till Select a workspace combo box and press TAB and then SHIFT + TAB.

## Expected Experience
When the keyboard focus is on any list item in the "Select an Azure ML job Schema" list and the Shift + TAB key is pressed, keyboard focus moved to "Select an Azure ML job Schema" combo box but JAWS/NVDA should announce about combo box. 

## User Impact:
Screen reader user will get confused if they press TAB key and screen reader is not announcing anything and keyboard focus is not visible and then pressing SHIFT + TAB also Screen reader is not announcing anything.

## Attachment

NVDA:

https://github.com/user-attachments/assets/b94b2e42-f1a7-4109-831e-91b9f7b8af53

JAWS:

https://github.com/user-attachments/assets/34b07540-b271-4397-9548-cf491cc328cc

Code snippet

![Image](https://github.com/user-attachments/assets/38f6de5a-dcc4-4a80-9944-dfd304331e88)

Similar issue of scenario 1 and 2

![Image](https://github.com/user-attachments/assets/2ca882e4-176d-4439-ba11-61fecb1b3b11)

**Comment by meganrogge** [comment]
Similar to the other issue #245219, I think this is because it lacks a placeholder. Pls create an issue for that extension. cc @TylerLeonhardt in case he disagrees here.

**Comment by TylerLeonhardt** [comment]
This is because we have the list focused by default (well, an item in the list) via `aria-activedescendant`. This is because an item is focused for quick usage of hitting ENTER when it shows. We did add a feature that lets you SHIFT+TAB to clear the selection in the list... the intent is that this would read the placeholder and title. Let me know if this does not do that.

Nevertheless, this quick pick is not following our guidelines:
https://code.visualstudio.com/api/ux-guidelines/quick-picks

because there is no placeholder. Please have the extension consult these.

**Comment by kapilvaishna** [comment]
@TylerLeonhardt You mentioned that pressing SHIFT + TAB would read the placeholder and title. However, in this bug title is given as "input", but the screen reader remains silent and does not announce anything, which creates an issue for screen reader users.

Could you please reopen the bug or provide your input.

**Comment by TylerLeonhardt** [comment]
Sorry @kapilvaishna, I forgot that we got rid of that in favor of the behavior of just hitting any key, like left arrow, will remove `aria-activedescendant`. In other words, so long as the key you hit doesn't influence the list show, it will remove `aria-activedescendant` and focus will be in the InputBox.

**Comment by meganrogge** [comment]
In order for the extension to fix this, they need to add a placeholder AFAICT. @TylerLeonhardt , pls correct me if i'm misunderstanding.

**Comment by TylerLeonhardt** [comment]
Yeah the extension needs a placeholder for sure. Please follow the guidelines: https://code.visualstudio.com/api/ux-guidelines/quick-picks

**Comment by kapilvaishna** [comment]
@TylerLeonhardt  I tried to check this issue where the placeholder is defined, but when pressing SHIFT + TAB, the keyboard shifts to the edit field while the screen reader remains silent. Then pressing SHIFT moves the screen reader focus to the edit field.

https://github.com/user-attachments/assets/e687e038-f7cf-4c73-af25-533256377d9a

**Comment by meganrogge** [comment]
@kapilvaishna I cannot repro on mac @TylerLeonhardt do you know why this would be different on windows? I can reproduce that.

https://github.com/user-attachments/assets/02060fe0-4f1f-40e4-ab8b-d66457c17594

**Comment by meganrogge** [comment]
Closing because a new one will be created to reset the clock

**Comment by kapilvaishna** [comment]
Closing the bug as logged new bug https://github.com/microsoft/vscode/issues/251068

**Comment by Test7987** [comment]
#Closed;
